### PR TITLE
Bump PG14 and PG15 versions for CI tests

### PR DESCRIPTION
**Pull request by naisila**
Postgres got minor updates yesterday, Aug10, this starts using the images with the latest
version for our tests, namely 14.9 and 15.4.

Depends on https://github.com/citusdata/the-process/pull/147

Issues: https://github.com/citusdata/citus/issues/7119

### CODE REVIEW

**Review by aykut-bozkurt — APPROVED** [review]
LGTM. Please do not forget to change the image tag in circleci/config.yml before merging.

**Comment by JelteF** [comment]
IMHO we can comment out the broken test described in #7119  for now. And then we can merge this after the the-process PR is merged.

File: .circleci/config.yml
Comment (JelteF) [inline .circleci/config.yml:9]: Reminder to not forget to change this to non dev image once the the-process PR is merged.

**Comment by naisila** [comment]
> IMHO we can comment out the broken test described in https://github.com/citusdata/citus/issues/7119 for now. And then we can merge this after the the-process PR is merged.

@onderkalaci what do you think?

**Comment by onderkalaci** [comment]
> > IMHO we can comment out the broken test described in #7119 for now. And then we can merge this after the the-process PR is merged.
> 
> @onderkalaci what do you think?

sounds good to me

**Pull request merged**
